### PR TITLE
Hac-4850 navigation edit

### PR DESCRIPTION
**Pull request by Randy424**
* Overview component becomes "Dashboard"
* The Overview nav is reassigned to our new product page.
* Nav item order changes

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@Randy424 I am just gonna look at the tickets to get some context and reference for the changes. There is quite a lot of changes and I want to be sure we are good.

**Comment by Randy424** [comment]
@Hyperkid123 Sorry for the confusion, my prettier setting is automated and it looks like it has made the git diff much harder to review. I will try to correct this with another commit.

**Comment by Hyperkid123** [comment]
Ah OK. This makes way more sense to me now. I just looked at the number of changes and got scared.

**Review by Hyperkid123 — APPROVED** [review]

File: static/stable/stage/navigation/openshift-navigation.json [hunk 1]
```diff
@@ -17,16 +36,6 @@
             "title": "Learning Resources",
             "href": "/openshift/learning-resources"
         },
-        {
-            "id": "overview",
-            "appId": "openshift",
-            "href": "/openshift/overview",
-            "title": "Overview",
-            "icon": "OpenShiftIcon",
```
Comment (KevinFCormier) [inline static/stable/stage/navigation/openshift-navigation.json:25]: @Randy424 I'm not sure what this `icon` property is used for, but you did remove it. Do you know?

Comment (epwinchell) [reply static/stable/stage/navigation/openshift-navigation.json:25]: @KevinFCormier The icons are a new addition to the Services dropdown menu in Chrome.

**Pull request merged**